samples/tfm_integration/psa_level_1/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+# Override the binary used by qemu
+set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_qemu.hex")
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+
+# Add "tfm" as an external project via the TF-M module's cmake file
+trusted_firmware_build(BINARY_DIR ${CMAKE_BINARY_DIR}/tfm
+                       BOARD ${TFMBOARD}
+                       CFGFILE "ConfigRegressionIPC"
+                       OUT_VENEERS_FILE VENEERS_FILE
+                       OUT_INCLUDE_PATH TFM_INCLUDE_PATH)
+
+project(tfm_psa_level_1)
+
+# Set "tfm" as a dependency so that it's built before the app
+add_dependencies(app tfm)
+
+# Source files in this sample
+target_sources(app PRIVATE src/main.c)
+target_sources(app PRIVATE src/psa_attestation.c)
+target_sources(app PRIVATE src/psa_crypto.c)
+target_sources(app PRIVATE src/shell.c)
+target_sources(app PRIVATE src/tfm_ipc.c)
+target_sources(app PRIVATE src/util_app_cfg.c)
+target_sources(app PRIVATE src/util_app_log.c)
+target_sources(app PRIVATE src/util_sformat.c)
+
+# Include "/install/export/tfm/include" when building
+target_include_directories(app PRIVATE ${TFM_INCLUDE_PATH})
+# Account for "psa/crypto_compat.h" not being in installs until fixed
+target_include_directories(app PRIVATE ${ZEPHYR_TFM_MODULE_DIR}/trusted-firmware-m/interface/include)
+
+# Include TF-M secure service source files
+set(TFM_ROOT_PATH ${CMAKE_BINARY_DIR}/tfm/install/export/tfm/src/)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_crypto_ipc_api.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_initial_attestation_ipc_api.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_its_ipc_api.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_ns_interface.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_platform_ipc_api.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_psa_ns_api.c)
+target_sources(app PRIVATE ${TFM_ROOT_PATH}/tfm_sst_ipc_api.c)
```

Comment:
To be update similar to this: https://github.com/zephyrproject-rtos/zephyr/pull/19985/commits/eb3c635377dda57e7f4c2aca83574465d4f5b595
and:
https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/6/commits/55f5ece809871bfd9756754da63faa557eb93d33

Reply:
See: https://github.com/zephyrproject-rtos/zephyr/pull/19985/commits/532792f8490244e54a274b1834cb919401c62fe6

Verified with `$ west build -p -b mps2_an521_nonsecure zephyr/samples/tfm_integration/psa_level_1/ -t run`